### PR TITLE
docs: update `README` of the plugin

### PR DESCRIPTION
**Pull request by BryanttV**
### Description
This PR updates the documentation of the plugin.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,74 +1,219 @@
...
+- GET ``/<lms_host>/platform-plugin-elm-credentials/<course_id>/api/credential-builder/``:
+  List all the topics in the course.
+
+  **Path parameters**
+
+  - ``course_id``: ID of the course.
+
+  **Query parameters**
+
+  - ``expired_at`` (optional): Date of the expiration of the credential. It
```
Comment (mariajgrimaldi) [inline README.rst]: It should be expires_at in the present simple

File: README.rst [hunk 2]
```diff
@@ -1,74 +1,219 @@
...
 
-Our real-time conversations are on Slack. You can request a `Slack
-invitation`_, then join our `community Slack workspace`_.
+  - ``client_id``: Client ID of the OAuth2 application. You can find it in the
+    Django admin panel. Normally, it is ``login-service-client-id``.
+  - ``grant_type``: Grant type of the OAuth2 application. Normally, it is
+    ``password``.
+  - ``username``: Username of the user.
+  - ``password``: Password of the user.
+  - ``token_type``: Type of the token. By default, it is ``bearer``
```
Comment (mariajgrimaldi) [inline README.rst:134]: You should also mention that a new oauth2 application can be used

File: README.rst [hunk 3]
```diff
@@ -1,74 +1,219 @@
...
+
+  **Query parameters**
+
+  - ``expired_at`` (optional): Date of the expiration of the credential. It
+    must be in the format ``YYYY-MM-DDTHH:mm:ss±hh:mm``. This parameter
+    modifies the properties ``validUntil`` and ``expirationDate`` of the ELMv3
+    format. By default, it is ``None``.
+  - ``to_file`` (optional): If it is ``true``, the response will be a JSON file
+    with the ELMv3 format. If it is ``false``, the response will be a JSON with
+    the ELMv3 format. By default, it is ``false``.
```
Comment (mariajgrimaldi) [inline README.rst]: I thought the default was true: https://github.com/eduNEXT/platform-plugin-elm-credentials/pull/2/files#diff-8e3405e8d75b7e762c648320762e1273966fa9ecab462fbde2514890e7419786R19

File: README.rst [hunk 4]
```diff
@@ -1,74 +1,219 @@
...
+
+    {
+      ...
+      "ELM_CREDENTIALS_DEFAULTS": {
+        "primary_language_code": "POR",
+        "primary_language_map": {"fr": "FRA", "de": "DEU"},
+        "org_country_code": "PRT",
+        "issuer_id": "8fb2a434-e46b-4fd9-80d1-ce7ff11c0048"
+      }
+    }
```
Comment (mariajgrimaldi) [inline README.rst:202]: Please list which settings I can change, what each means, and what the Unidigital project defaults are.

**Review by mariajgrimaldi — CHANGES_REQUESTED** [review]
I left a few comments for you to address. Also, can you add how to test the JSON file with the EU credentials builder? Thanks

File: README.rst [hunk 5]
```diff
@@ -1,74 +1,219 @@
...
-|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
-|license-badge| |status-badge|
+|ci-badge| |license-badge| |status-badge|
 
 Purpose
 *******
 
-Open edx plugin that includes API to generate JSON files in ELMv3
+Open edX plugin that includes an API endpoint to generate a JSON file with the
+European Learning Model (ELMv3) data model format. Only learners who have
```
Comment (mariajgrimaldi) [inline README.rst]: can you link documentation?

File: README.rst (hunk 1 above)
Comment (BryanttV) [reply README.rst]: Done

File: README.rst (hunk 2 above)
Comment (BryanttV) [reply README.rst:134]: Done

File: README.rst (hunk 3 above)
Comment (BryanttV) [reply README.rst]: Yes, the default is true, I fixed it

File: README.rst (hunk 4 above)
Comment (BryanttV) [reply README.rst:202]: Done

File: README.rst (hunk 5 above)
Comment (BryanttV) [reply README.rst]: Done

File: README.rst [hunk 6]
```diff
@@ -1,74 +1,252 @@
...
+
+Configuring plugin settings
+***************************
+
+You can configure some settings for the plugin. The available settings are the
+following:
+
+* **primary_language_code**: Primary language code of the credential. It is
+  used to generate the ``primaryLanguage`` property of the ELMv3 format. By
+  default, the language defined in the platform is used.
```
Comment (mariajgrimaldi) [inline README.rst]:
```suggestion
  default, the language defined by the Django setting `LANGUAGE_CODE` is used.
```

File: README.rst [hunk 7]
```diff
@@ -1,74 +1,252 @@
...
+* **primary_language_map**: Map of the primary language code with the language.
+  This settings is useful when the language defined in the primary language
+  code is different from English or Spanish.
+* **org_country_code**: Country code of the organization. It is used to
+  generate the ``countryCode`` property of the ELMv3 format. By default, it is
+  ``ESP``.
+* **issuer_id**: ID of the issuer. It is used to generate the ``issuer`` property
+  of the ELMv3 format. By default, a random UUID is generated.
```
Comment (mariajgrimaldi) [inline README.rst]:
```suggestion
  of the ELMv3 format. By default, a random in-memory UUID is generated.
```

File: README.rst [hunk 8]
```diff
@@ -1,74 +1,252 @@
...
+   .. code-block:: python
+
+    ELM_CREDENTIALS_DEFAULTS = {
+      "primary_language_code": "POR",
+      "primary_language_map": {"fr": "FRA", "de": "DEU"},
+      "org_country_code": "PRT",
+      "issuer_id": "8fb2a434-e46b-4fd9-80d1-ce7ff11c0048",
+    }
+
+The order of precedence is the following:
```
Comment (mariajgrimaldi) [inline README.rst:215]:
```suggestion
The order of precedence when looking for the defaults is the following:
```

**Review by mariajgrimaldi — APPROVED** [review]

**Pull request merged**